include/network/networkinterfacemanager.hpp

```diff
@@ -7,10 +7,106 @@
 #ifndef NETWORKINTERFACEMANAGER_HPP_
 #define NETWORKINTERFACEMANAGER_HPP_
 
+#include <optional>
```

Comment:
Let's rename file to `interfacemanager.hpp`

Reply:
Done